qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -393,6 +398,12 @@ def run(
         """
         experiment_data = super().run(backend, analysis, **run_options)
 
+        experiment_data.metadata["calibrations"] = {
+            "calibrations": self._cals.serialize(),
```

Comment:
When I said I was tempted to make saving the parameters the default, I had been thinking of saving all the current parameter values, a data set which would have a fixed size of data for a device with a set number of qubits. With this, one could reload from a previous cal and continue working on a device from where one left off. The serialized Calibrations data will include the history of all the parameters and will grow continuously as more calibrations are run. Saving that for each experiment might be a bit much. What do you think?

Longer term, I think it would be a good idea to split the roles of managing the currently calibrated values (`Calibrations` class) and tracking the history of calibrated values (`CalibrationsStore` class), with helper functions for resetting the currently loaded values to a query of the history store (cutoff, quality, group, etc.).

Reply:
That sounds good. How about: f8bd843